sympy/physics/continuum_mechanics/beam.py

```diff
@@ -0,0 +1,343 @@
+        this method would represent the PointLoads and DistributedLoads using
+        Singularity Functions.
+
+        Examples
+        ========
+
+        """
+
+
+class PointLoad(object):
+    """A Point Load.
+
+    A load applied to a single, specific point. It is also known as a
+    concentrated load.
+
+    Parameters
+    ==========
+    location : Point
+        The specific point of the applied load.
+    value : Sympifyable
+        A SymPy expression representing the value of the applied load.
+
+    Examples
+    ========
+    >>> from sympy.physics.continuum_mechanics.beam import PointLoad
+    >>> from sympy.physics.mechanics import Point
+    >>> p = Point('4')
+    >>> PointLoad(location = p, value = -4)
+    PointLoad(4, -4, Load)
+
+    A Moment can be defined just by passing moment=True as an argument.
+
+    >>> PointLoad(location = p, value = -4, moment=True)
+    PointLoad(4, -4, Moment)
+
+    """
+
+    def __init__(self, location, value, moment=False):
+        self._location = location
+        self._value = value
+        self._moment = moment
+
+    def __str__(self):
+        if self.moment:
+            str_sol = 'PointLoad(%s, %s, %s)' % (sstr(self.location), sstr(self.value), sstr('Moment'))
+            return str_sol
+        str_sol = 'PointLoad(%s, %s, %s)' % (sstr(self.location), sstr(self.value), sstr('Load'))
+        return str_sol
+
+    __repr__ = __str__
+
+    @property
+    def location(self):
+        """Location of the applied Point Load."""
+        return self._location
+
+    @location.setter
+    def location(self, l):
+        if not isinstance(l, Point):
+            raise TypeError("PointLoad location attribute must be a Point object.")
+        self._location = l
+
+    @property
+    def value(self):
+        """Value of the applied Point Load. """
+        return self._value
+
+    @value.setter
+    def value(self, v):
+        self._value = sympify(v)
+
+    @property
+    def moment(self):
+        """Stores whether a Point Load is a load or a couple. """
+        return self._moment
+
+    @moment.setter
+    def moment(self, m):
+        if not isinstance(m, bool):
+            raise TypeError("PointLoad moment attribute must be a bool object.")
+        self._moment = m
+
+
+class DistributedLoad(object):
+
+    """A Distributed Load.
+
+    A load applied across a length instead of at one point.
+
+    Parameters
+    ==========
+    start : Point
+        The starting point of the applied load.
+    order : Sympifyable
+        The order of the applied load.
+    value : Sympifyable
+        A SymPy expression representing the value of the applied load.
+
+    Examples
+    ========
+    >>> from sympy.physics.continuum_mechanics.beam import DistributedLoad
+    >>> from sympy.physics.mechanics import Point
+    >>> from sympy import Symbol
+    >>> a = Point('4')
+    >>> b = 2
+    >>> DistributedLoad(start = a, order = b, value = 2)
```

Comment:
Are you going to support start and end with this where it creates two distributed loads, one that makes a composite of two?